settings.py

```diff
@@ -21,6 +21,6 @@
 
 USE_I18N = True
 
-USE_TZ = False
+USE_TZ = True
 
-INSTALLED_APPS = ("db",)
+INSTALLED_APPS = ("db", "django_extensions",)
```

Comment:
why did you add this?)

Reply:
It helped me to check the code step by step during development, now I will clean it up)